tutor/templates/apps/openedx/settings/partials/common_all.py

```diff
@@ -246,5 +246,14 @@
     "user": None,
 }
 
+LEARNING_CORE = {
```

Comment:
Actually, I should probably make this `OPENEDX_LEARNING`, to reduce confusion...